src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "3.0.0b9"
+VERSION = "3.0.0b11"
```

Comment:
```suggestion
VERSION = "3.0.0b10"
```
b10 is not released yet